CASE-STUDIES-AND-EXPERIENCES.md

```diff
@@ -5,23 +5,16 @@ title: Case Studies and Experiences
 site_side: true
 tags: security-qualitative-metrics
 project: true
-level: 4
 type: documentation
 pitch: The OWASP Security Qualitative Metrics is the most detailed list of metrics which evaluate security level of web projects. It shows the level of coverage of OWASP ASVS. 
 ---
 
+# ![Project Logo](assets/images/logo3_small.png) OWASP Security Qualitative Metrics
 
-
-# ![Project Logo](images/logo3_small.png) Case Studies and Experiences 
-# [![OWASP Flagship](https://img.shields.io/badge/owasp-flagship-blue.svg)](https://owasp.org/projects/)
- [![GitHub license](https://img.shields.io/github/license/Naereen/StrapDown.js.svg)](https://github.com/OWASP/www-project-security-qualitative-metrics/blob/master/LICENSE)
-
- [![GitHub release](https://img.shields.io/github/release/Naereen/StrapDown.js.svg)](https://github.com/OWASP/www-project-security-qualitative-metrics/releases)
```

Comment:
Wrong repo/project. This/your project doesnt have a 1.1.1 release

Reply:
I had some difficulty on setting up a version. If you can correct it. You are welcome to do it.

Reply:
The correction is in this PR, just merge it